.evergreen/config_generator/components/sasl/openssl.py

```diff
@@ -33,15 +33,19 @@
     ('ubuntu1604',        'clang',      None, ['cyrus']),
     ('ubuntu1804-arm64',  'gcc',        None, ['cyrus']),
     ('ubuntu1804',        'gcc',        None, ['cyrus']),
+    ('ubuntu2004-arm64',  'gcc',        None, ['cyrus']),
     ('ubuntu2004',        'gcc',        None, ['cyrus']),
     ('windows-vsCurrent', 'vs2017x64',  None, ['cyrus']),
 ]
 
 TEST_MATRIX = [
     ('rhel81-power8',     'gcc',       None, 'cyrus', ['auth'], ['server',          ], [       '4.2', '4.4', '5.0', '6.0', '7.0', 'latest']),
     ('rhel83-zseries',    'gcc',       None, 'cyrus', ['auth'], ['server',          ], [                     '5.0', '6.0', '7.0', 'latest']),
-    ('ubuntu1804-arm64',  'gcc',       None, 'cyrus', ['auth'], ['server',          ], [       '4.2', '4.4', '5.0', '6.0', '7.0', 'latest']),
-    ('ubuntu1804',        'gcc',       None, 'cyrus', ['auth'], ['server', 'replica'], ['4.0', '4.2', '4.4', '5.0', '6.0', '7.0', 'latest']),
+    ('ubuntu1804-arm64',  'gcc',       None, 'cyrus', ['auth'], ['server',          ], [       '4.2', '4.4', '5.0', '6.0',                ]),
+    ('ubuntu1804',        'gcc',       None, 'cyrus', ['auth'], ['server', 'replica'], ['4.0', '4.2', '4.4', '5.0', '6.0',                ]),
+    # Test 7.0+ with Ubuntu 20.04+ since MongoDB 7.0 no longer ships binaries for Ubuntu 18.04.
+    ('ubuntu2004-arm64',  'gcc',       None, 'cyrus', ['auth'], ['server',          ], [                                   '7.0', 'latest']),
+    ('ubuntu2004',        'gcc',       None, 'cyrus', ['auth'], ['server',          ], [                                   '7.0', 'latest']),
```

Comment:
Since these entries are not grouped together, suggest simplifying the formatting.

```python
    ('ubuntu1804',        'gcc',       None, 'cyrus', ['auth'], ['server', 'replica'], ['4.0', '4.2', '4.4', '5.0', '6.0',                ]),

    # Test 7.0+ with Ubuntu 20.04+ since MongoDB 7.0 no longer ships binaries for Ubuntu 18.04.
    ('ubuntu2004-arm64',  'gcc',       None, 'cyrus', ['auth'], ['server'], ['7.0', 'latest']),
    ('ubuntu2004',        'gcc',       None, 'cyrus', ['auth'], ['server'], ['7.0', 'latest']),
    ('windows-vsCurrent', 'vs2017x64', None, 'cyrus', ['auth'], ['server'], [       'latest']),
```